milestone2.ts

```diff
@@ -139,23 +154,27 @@ type UpdateVolunteer = (
   vol: Volunteer,
   updates: Partial<Volunteer>
 ) => Volunteer;
-let updateVolunteer: UpdateVolunteer; // Code here
+let updateVolunteer: UpdateVolunteer = (vol,update) =>
+{
+  vol = {...vol,...update};
+  return vol;
+}
 
-// let kyleNew = updateVolunteer(kyleClone);
-//console.log(kyleNew);
+kyleNew = updateVolunteer(kyleClone,updatedLocation);
+console.log(kyleNew);
 
 // Question 7: Object Destructuring
 /* Now that we have our updated kyle, lets use object destructuring to get his name, age, and city
  */
-
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
+  let name, age, city; 
+  [name,age,city] = [vol.name,vol.age,vol.city];
```

Comment:
It looks like you didn't use object destructuring here, could you resubmit this question with object destructuring to get those fields from vol? If you are confused about object destructuring, you can find an example in the demo from [Bootcamp Session 3](https://h4i.notion.site/Intro-to-Javascript-83f29480d7c14b0c9163212b96f6ab0f)

Reply:
Hey,

Sorry for the delay...I added object destructuring to question 7 now.